src/schema/nmdc.yaml

```diff
@@ -638,9 +639,13 @@ enums:
       QC non-rRNA R1:
```

Comment:
this is pretty good in general. Ideally each permissible value would have a description and an annotation showing legal file name patterns